### DISCUSSION TITLE
특정 웹 페이지 로딩이 매우 느립니다.

### DISCUSSION THREAD

**Discussion by linuxias** [pronist/hELLO#129] [Q&A] [answered]
**환경 (버그가 발생한 환경을 적어주세요.):**

- 운영체제: window 10, ubuntu 16.04
- 브라우저: whale
- 버전: [e.g. 22]
- 스킨 버전 (스킨 버전은 index.xml 에서 확인): [e.g 3.4.0]

**설명**
간혹 특정 페이지에서 로딩 시간이 매우 깁니다. 해당 스킨 적용 후 몇몇 페이지의 로딩이 1분 이상 넘어가는 경우도 발생합니다.

**발생 과정**
페이지 접속 시 로딩

**스크린샷 및 링크**
https://sonseungha.tistory.com/553?category=913655

**Comment by pronist** [comment, marked as answer]
@linuxias 안녕하세요!

특정 웹페이지의 로딩이 느린경우 여러가지 원인이 있을 수 있습니다.

- 글에 포함된 이미지 원본이 너무 큰 커서 전송 데이터가 큰 경우.
- GIF 가 많은 경우.
- 광고 스크립트 또는 별도로 로드한 스크립트가 올바르게 로드 되지 않았거나 문제가 발생한 경우.

대체로 이미지와 관련된 문제로 인해 로딩이 느려지는 경우가 많습니다. 이 점 점검해보시기 바랍니다.
